doc/basics.md

```diff
@@ -399,6 +399,32 @@ And another one:
 
 Notice how we can string several operation (`parser.parse(abstract_json)["str"]["123"]["abc"].get<double>()`) and only check for the error once, a strategy we call  *error chaining*.
 
+
+The next two functions will take as input a JSON document containing an array with a single element, either a string or a number. They return true upon success.
+
+```C++
+simdjson::dom::parser parser{};
+
+bool ParseDouble(const char *j, double &d) {
```

Comment:
... at some point I think I'm going to go in and rename this to parse_double, if you don't, just to match the sensibilities of the rest of the project :)